freebridgefe/.env.example

```
@@ -0,0 +1,6 @@
+# Backend API base URL
+VITE_API_BASE_URL=http://localhost:8080
+
+# PortOne V2 channel configuration
+VITE_PORTONE_STORE_ID=store-6744519b-003b-412d-803e-2762e616152e
+VITE_PORTONE_CHANNEL_KEY=channel-key-df065c76-31d3-4f99-823a-10d1e35f93dd
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_PORTONE_CHANNEL_KEY key should go before the VITE_PORTONE_STORE_ID key

(UnorderedKey)